test/e2e/common/reporter/junit.go

```diff
@@ -0,0 +1,19 @@
+package reporter
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+)
+
+func GetReporters(name string) []Reporter {
+	reportDir := os.Getenv("e2e_reports_dir")
+	if reportDir == "" {
+		panic("reportDir not defined")
```

Comment:
```suggestion
		panic("reportDir not defined - define via e2e_reports_dir environment variable")
```

reportDir is internal representation, for user-experience it's better to notice what needs to be changed w/o llooking into the code.